sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -1231,8 +1364,10 @@ case class Decode(bin: Expression, charset: Expression)
  * (one of 'US-ASCII', 'ISO-8859-1', 'UTF-8', 'UTF-16BE', 'UTF-16LE', 'UTF-16').
  * If either argument is null, the result will also be null.
  */
+// scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(str, str) - Encode the first argument using the second argument character set.")
+  usage = "_FUNC_(str1, str2) - Encodes the first argument using the second argument character set.")
```

Comment:
`_FUNC_(str1, str2) - Encodes the first argument using the second argument character set.`
->
```
_FUNC_(expr, charset) - Encodes the string `expr` using the character set `charset`.